modelator/src/model/checker/apalache/mod.rs

```diff
@@ -72,12 +72,25 @@ impl Apalache {
         // Gets Apalache command with tdir as working dir
         let cmd = apalache_start_cmd(&tdir, runtime);
 
+        // Check if the main tla module contains a View
+        // The view will have a generated name 'ViewForTestNeg'
+        // If it has one, then use it.
+        let view = match input_artifacts
+            .tla_file
+            .file_contents_backing()
+            .contains("ViewForTestNeg")
+        {
```

Comment:
you can use [`bool::then()`](https://doc.rust-lang.org/std/primitive.bool.html#method.then) here. :)

Reply:
Nice, thanks